frontend/app/src/premium/premium.ts

```diff
@@ -171,5 +171,6 @@ declare global {
     'chartjs-plugin-zoom': any;
     zod: any;
     bn: any;
+    'vue-i18n': any;
```

Comment:
Do we really need to expose the whole module to the components? isn't the i18n object/interface enough?

Reply:
For this part I need the module to declare type `$t and $tc` on premium-components <template>

```
declare module 'vue/types/vue' {
  interface Vue {
    $t: typeof VueI18n.prototype.t;
    $tc: typeof VueI18n.prototype.tc;
  }
}
```

Reply:
In the templates? just use the exposed api wrappers I suggested, `$t` and `$tc` are going away soon, initially, we will update to the [composable](https://github.com/intlify/vue-i18n-composable) in preparation of `vue 3`

Reply:
In vue 3/vue-18n the approach would be to do `const { t, tc } = useI18n();`